polar-core/src/bindings.rs

```diff
@@ -787,37 +780,6 @@ mod test {
         }
     }
 
-    #[test]
-    fn deref() {
```

Comment:
This test also covers some other logic in the binding manager, can we keep it but just replace `deref` with `deep_deref`?

Reply:
👍